source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -1,203 +1,202 @@
 
 It's recommended to also review the list of :doc:`known issues <known-issues>`.
 
 ## Importing Projects from Previous Years
 
-Due to internal GradleRIO changes, it is necessary to update projects from previous years. After :doc:`Installing WPILib for 2024 </docs/zero-to-robot/step-2/wpilib-setup>`, any 2023 projects must be :doc:`imported </docs/software/vscode-overview/importing-last-years-robot-code>` to be compatible.
+Due to internal GradleRIO changes, it is necessary to update projects from previous years. After :doc:`Installing WPILib for 2025 </docs/zero-to-robot/step-2/wpilib-setup>`, any 2024 projects must be :doc:`imported </docs/software/vscode-overview/importing-last-years-robot-code>` to be compatible.
 
 ## Major Changes (Java/C++)
 
 These changes contain *some* of the major changes to the library that it's important for the user to recognize. This does not include all of the breaking changes, see the other sections of this document for more changes.
 
-- Added support for :doc:`XRP robots </docs/xrp-robot/index>`
-- Projects now default to supporting Java 17 features
-- Multiple NetworkTables networking improvements for improved reliability and robustness and structured data support using protobuf
-- Java now uses the Serial GC by default on the roboRIO; this should improve performance and reduce memory usage for most robot programs
-- Performance improvements and reduced worst-case memory usage throughout libraries
-- Added a typesafe unit system for Java (not used by the main part of WPILib yet)
-- Disabled LiveWindow in Test Mode by default. See See :ref:`docs/software/dashboards/smartdashboard/test-mode-and-live-window/enabling-test-mode:Enabling LiveWindow in Test Mode` to enable it.
-- SysId has been rewritten to remove project generation; Replaced with data logging within team robot program
+- Added :doc:`annotation based logging (Epilogue) </docs/software/telemetry/robot-telemetry-with-annotations>` for Java
+- The :doc:`Java units library </docs/software/basic-programming/java-units>` has been refactored to have unit-specific measurement classes instead of a single generic ``Measure`` class. The new measurement classes have clearer names (``Distance`` instead of ``Measure<Distance>``, or ``LinearAcceleration`` instead of ``Measure<Velocity<Velocity<Distance>>>``), and implement math operations to return the most specific result types possible instead of a wildcard ``Measure<?>``.
+- Add :doc:`persistent alerts API </docs/software/telemetry/persistent-alerts>`. Alerts are displayed on supported dashboards such as Shuffleboard and Elastic.
+- Add LED pattern API for easily animating addressable LEDs
+- Java Source and Target compatibility have been bumped to JDK 17
```

Comment:
Should we note that there's no runtime changes here? We started deploying a Java 17 VM last season, but defaulted to compiling for 11 in case we needed to downgrade.

Reply:
Yeah, I struggled with what to say here. Really the only impact is for people that did downgrade to the 11 VM